addons/hw_drivers/tools/helpers.py

```diff
@@ -273,11 +273,27 @@ def get_odoo_server_url():
 def get_token():
     return read_file_first_line('token')
 
-def get_version():
+
+def get_commit_hash():
+    return subprocess.check_output(
+        ['git', '--work-tree=/home/pi/odoo/', '--git-dir=/home/pi/odoo/.git', 'rev-parse', '--short', 'HEAD']
+        ).decode('ascii').strip()
+
+
+def get_version(detailed_version=False):
     if platform.system() == 'Linux':
-        return read_file_first_line('/var/odoo/iotbox_version')
+        image_version = read_file_first_line('/var/odoo/iotbox_version')
     elif platform.system() == 'Windows':
-        return 'W22_11'
+        # updated manually when big changes are made to the windows virtual IoT
+        image_version = '22.11'
+
+    version = platform.system()[0] + image_version
```

Comment:
Maybe useless because the versions are not written dynamically but manually

Reply:
what do you mean by that ? the value will still be to be edited manually (just no longer need to W prefix it)